package.json

```diff
@@ -1,108 +1,43 @@
 {
-  "name": "fd-vue-webapp",
-  "version": "1.0.0",
-  "description": "A Vue.js web application for Freedomotic",
-  "author": "Freedomotic Team",
+  "name": "freedomotic",
+  "version": "0.0.1",
+  "description": "A Quasar Framework app",
+  "productName": "Freedomotic Web App",
+  "cordovaId": "org.cordova.quasar.app",
+  "author": "atanda rasheed <atandarash@gmail.com>",
   "private": true,
   "scripts": {
-    "dev": "node build/dev-server.js",
-    "start": "node server.js",
-    "heroku-postbuild": "npm install --only=dev --no-shrinkwrap && npm run build",
```

Comment:
why are you removing these scripts?

Reply:
Those were removed because you need to run the app using the globally installed **quasar-cli**, but I will restore the **heroku-postbuild** script

Reply:
ok :smiley: